docs/operations.md

```diff
@@ -36,11 +40,30 @@ metadata:
-
-After checking the Secret, the reconciler prepares the module's chart for provisioning. It adds `app.kubernetes.io/managed-by: btp-manager` label to all chart resources, sets data from the required Secret as overrides and applies them among overrides from `values.yaml`. When the chart install info is correct, the reconciler starts the provisioning and waits specified time for all chart resources to be in `Ready` state. If timeout is reached, the CR receives `Error` state and the resources are checked again in the next reconciliation. The reconciler has a fixed set of [timeouts](../operator/controllers/btpoperator_controller.go) defined as `consts` which limit the processing time for performed operations. The provisioning is successful when all chart resources are in `Ready` state and this is the condition which allows the reconciler to set the CR in `Ready` state.
+The BtpOperator reconciler picks up the created CR and determines whether it should be responsible for representing the
+module status. The BtpOperator CR reflects the status of the operand, that is, SAP BTP Service Operator, only when it is
+the oldest CR present in the cluster. In that case a finalizer is added, the CR is set to `Processing` state and the
+reconciliation proceeds.
+Otherwise, it is given an `Error` state with the condition reason `OlderCRExists` and message containing details
+about the CR responsible for reconciling the operand.
+
+Next, the reconciler looks for a `sap-btp-manager` Secret in the `kyma-system` Namespace. This Secret contains Service
+Manager credentials for SAP BTP Service Operator and should be delivered to the cluster by KEB. If the Secret is
+missing, an error is thrown, the reconciler sets `Error` state (with the condition reason `MissingSecret`) in the CR and stops the reconciliation until the Secret
+is created. When the Secret is present in the cluster, the reconciler verifies whether it contains required data. The
+Secret should contain the following keys: `clientid`, `clientsecret`, `sm_url`, `tokenurl`, `cluster_id`. None of the
+key values should be empty. If some required data is missing, the reconciler throws an error with the message about
+missing keys/values, sets the CR in `Error` state (reason `InvalidSecret` and stops the reconciliation until there is a change in the required
```

Comment:
Maybe it would be better to say (due to `InvalidSecret`), WDYT?

Reply:
`InvalidSecret` is the value of `reason` field in status structure. So due to missing keys/values CR is in the state `Error` with reason `InvalidSecret`.

Reply:
Right, so do not change it.